flycheck.el

```diff
@@ -4335,6 +4349,34 @@ universal prefix arg, and only the id with normal prefix arg."
       (seq-do #'kill-new (reverse messages))
       (message (string-join messages "\n")))))
 
+(defun flycheck-explain-error-at-point ()
+  "Display an explanation for the first error at point."
+  (interactive)
+  (let* ((errors-at-point (flycheck-overlay-errors-at (point)))
+         (first-error (car errors-at-point))
+         (checker (and first-error (flycheck-error-checker first-error)))
+         (explainer (and first-error (flycheck-checker-get checker
+                                                           'error-explainer)))
+         (explanation (and explainer (funcall explainer first-error))))
+    (cond
+     ((null first-error) (message "No errors at point"))
+     ((null explainer) (message (format "No explainer for %S checker" checker)))
```

Comment:
Do we need these messages?  Why not just show the explanation or nothing if there is none?